test/fixtures/models/katello_repositories.yml

```diff
@@ -458,6 +458,21 @@ busybox2:
   url:                  'http://docker.io'
   docker_upstream_name: 'busybox'
 
+busybox3:
+  name:                 busybox
+  pulp_id:              "Default_Organization-Test-busybox2-library"
+  content_id:           1
+  content_type:         docker
+  label:                ___busybox___
+  relative_path:        'ACME_Corporation/library/busybox'
+  environment_id:       <%= ActiveRecord::FixtureSet.identify(:library) %>
+  product_id:           <%= ActiveRecord::FixtureSet.identify(:puppet_product) %>
+  gpg_key_id:           <%= ActiveRecord::FixtureSet.identify(:fedora_gpg_key) %>
+  content_view_version_id: <%= ActiveRecord::FixtureSet.identify(:library_default_version) %>
+  unprotected:           <%= true %>
+  url:                  'http://docker.io'
+  docker_upstream_name: 'busybox'
+
```

Comment:
Lets not add an entire fixture just for this test.  Something like this should be sufficient:

repo = Repository.new(:name => 'some name')
repo.set_container_repository_name

We don't need to persist anything to the db to test this, for faster tests!